package.json

```diff
@@ -181,30 +180,32 @@
     "@types/superagent": "^4.1.24",
     "@types/supertest": "^2.0.16",
     "@types/uuid": "^9.0.7",
-    "@typescript-eslint/eslint-plugin": "^5.59.11",
-    "@typescript-eslint/parser": "^5.59.11",
+    "@typescript-eslint/eslint-plugin": "^6.16.0",
```

Comment:
These are all the same versions that PLP uses
I've only bumped the versions of eslint and prettier related stuff